modules/Bio/EnsEMBL/DBSQL/SequenceAdaptor.pm

```diff
@@ -80,64 +69,14 @@ our $SEQ_CACHE_MAX   = (2 ** $SEQ_CHUNK_PWR) * $SEQ_CACHE_SZ;
 =cut
 
 sub new {
-  my $caller = shift;
-
+  my ($caller, $db, $chunk_power, $cache_size) = shift;
```

Comment:
Oh. That's a bug surely. Should be assigned to the stack so we can set all of these values. Can we change it?